_includes/v3.6/charts/calico/values.yaml

```diff
@@ -1,13 +0,0 @@
-# Select which datastore mode. Can be 'etcd' or 'kdd'.
```

Comment:
Huh, was this just hanging around and not being used?

Reply:
Yup, It looks like I missed removing it when we moved the values to the ruby script